src/Autofac.Integration.Owin/OwinContextExtensions.cs

```diff
@@ -50,5 +50,22 @@ public static void SetAutofacLifetimeScope(this IOwinContext context, ILifetimeS
             context.Set(Constants.OwinLifetimeScopeKey, scope);
         }
 
+        /// <summary>
+        /// Removes the Autofac lifetime scope from the OWIN context if it is present.
+        /// </summary>
+        /// <param name="context">The OWIN context.</param>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown if <paramref name="context" /> is <see langword="null" />.
+        /// </exception>
+        /// <remarks>The caller is responsible for the appropriate disposal of the passed <see cref="ILifetimeScope"/>.</remarks>
```

Comment:
This 'remarks' block probably just needs removing.

Reply:
This is fixed by the latest commit